.env_sample

```diff
@@ -1,3 +0,0 @@
-DEVELOPMENT=1

```

Comment:
Why did you delete the file? Could you please add this back in as this is for all students? If you don't need it locally, you can delete it, but please don't commit the change.

Reply:
File added back in - resolving conversation